pkgs/tools/misc/lesspipe/default.nix

```diff
@@ -0,0 +1,34 @@
+{ stdenv, fetchFromGitHub, perl }:
+
+stdenv.mkDerivation rec {
+  name = "lesspipe";
```

Comment:
The version string needs to be embedded in the name; currently, `version` is not used for anything, somewhat perversely, the version is actually parsed out of `name` ...